src/Sylius/Bundle/UserBundle/Command/RoleCommandInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Command;
+
+use Sylius\Component\Core\Model\UserInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Loïc Frémont <loic@mobizel.com>
+ */
+interface RoleCommandInterface
```

Comment:
AbstractRoleCommand object use this interface. This command calls "executeRoleCommand" method. Thus, to extend AbstractRoleCommand, you need to add executeRoleCommand method.